.travis.yml

```diff
@@ -4,7 +4,7 @@ dist: xenial
 language: python
 env:
   global:
-    - LSR_ANSIBLES='ansible==2.6.* ansible==2.7.* ansible==2.8.*'
```

Comment:
The Ansible versions can be updated without discussion, therefore I added then in #179